src/utils/attachments.ts

```diff
@@ -0,0 +1,39 @@
+import { flatten } from "fp-ts/lib/Array";
+import { CreatedMessageWithContentAndAttachments } from "generated/backend/CreatedMessageWithContentAndAttachments";
+import { PrescriptionData } from "generated/backend/PrescriptionData";
+import { CreatedMessageWithContent } from "generated/io-api/CreatedMessageWithContent";
+import { ResponseSuccessJson } from "italia-ts-commons/lib/responses";
+import { toBarcode } from "./barcode";
+
+const toMessageAttachments = async (name: string, value: string) => {
```

Comment:
since this methods are not generic but tied to the barcode generation step, we must use a name (for the file as well) that reflect this link (ie. `toBarcodeAttachments`)